src/ray/stats/metric_exporter_client_test.cc

```diff
@@ -53,8 +53,6 @@ class MockExporterClient1 : public MetricExporterDecorator {
     RAY_LOG(DEBUG) << "Client 1 " << client1_count << " last metric "
                    << points.back().metric_name << ", value " << points.back().value;
     RecordLastHistData(points);
-    // Point size must be less than or equal to report batch size.
-    ASSERT_GE(kMockReportBatchSize, points.size());
```

Comment:
batching should be handled by the client now.

Reply:
I'm not sure why every client should know batch size of reporting. We'd better make code more clean and no replicated processing for different clients because MetricsExporter will collect points from view  data and MetricExporterClient just send them to desitination.